### PR TITLE
Add `tqdm` progress bars to various methods and commands

### PR DESCRIPTION
**Pull request by rohitg00**
* **`autorag/__init__.py`**
  - Import `tqdm` for progress bar support

* **`autorag/chunker.py`**
  - Add `tqdm` progress bar to `start_chunking` method

* **`autorag/cli.py`**
  - Add `tqdm` progress bar to `evaluate` command
  - Add `tqdm` progress bar to `run_api` command
  - Add `tqdm` progress bar to `run_web` command

### CODE REVIEW

**Comment by rohitg00** [comment]
Proposed changes for the feature request: https://github.com/Marker-Inc-Korea/AutoRAG/issues/878

File: autorag/chunker.py [hunk 1]
```diff
@@ -46,12 +47,13 @@ def start_chunking(self, yaml_path: str):
...
-			parsed_result=self.parsed_raw,
-			trial_path=os.path.join(self.project_dir, trial_name),
-		)
+		for _ in tqdm(range(1), desc="Chunking Progress"):
+			run_chunker(
+				modules=input_modules,
+				module_params=input_params,
+				parsed_result=self.parsed_raw,
+				trial_path=os.path.join(self.project_dir, trial_name),
+			)
```
Comment (vkehfdl1) [inline autorag/chunker.py:56]: I don’t figure out the reason why the tqdm range shown in here. It is not that different between `logger.info` about chunking done.

**Review by vkehfdl1 — CHANGES_REQUESTED** [review]
Also the other things.
Simply add tqdm range at the non-iteratable objects do not improve user experience.
I really appreciate your PR and your time to work on this. 
But we are working on a solution that are not using log. Because we think we are using tqdm and logger with `rich` pretty well along the cli.

However, as you feel the same, only using cli terminal and seeing the log is really hard to recognize the process. Actually we log all the info of the initializing, running, and deleting each module and tracking each module execution.
But yet, it is hard to recognize.

Maybe it is our fault that do not write an issue specification well.

Anyway, it is hard to accept this code for our main branch because it seems to add unmeaningful log at the terminal. 

Thanks for the interest to our project, and I hope we can find the good part to the contribution to our project.

**Comment by vkehfdl1** [comment]
close this at #903

**Pull request closed without merging**